src/requests/autosuggest-selection.ts

```diff
@@ -0,0 +1,18 @@
+import { fetchGet } from '../fetch'
+import { AutosuggestOptions, autosuggestOptionsToQuery } from './autosuggest'
+
+export const autosuggestSelection = (
+  rawInput: string,
+  selection: string,
+  rank: number,
+  options: AutosuggestOptions = {},
+  sourceApi: 'text' | 'voice' = 'text',
+): Promise<null> => {
+  return fetchGet("autosuggest-selection", {
+    ['raw-input']: rawInput,
+    selection,
+    rank: rank.toString(),
+    ['source-api']: sourceApi,
```

Comment:
Minor, but do you need the brackets around this and the one above?

Reply:
They are redundant, I've removed.